src/components/employees/SingleEmployee.jsx

```diff
@@ -3,57 +3,64 @@ import { Link, useParams } from 'react-router-dom';
 import { Box, Button, Container, Typography } from '@mui/material';
 import getLocationString from '../../utils/getLocationString';
 import useButtonDisplay from './hooks/useButtonDisplayType';
+import { useGetEmployeeByIdQuery } from '../../services/gym';
 
 
 const SingleEmployee = () => {
   const user = useSelector(state => state.user)
   const urlParams = useParams();
-  const employeesList = useSelector(state => state.employees) ;
-  const employee = employeesList.find(emp => {
-    return emp.id === parseInt(urlParams.id)
-  });
-
-  const {buttonDisplayType } = useButtonDisplay(employee, user);
+  const { data: employee } = useGetEmployeeByIdQuery(urlParams?.id)
 
+const {buttonDisplayType} = useButtonDisplay(employee, user);
 
   return (
-    <Box sx={{
-      bgcolor: theme => theme.palette.primary.main,
-      height: 'fit-content',
-      m: '8rem auto 0 auto',
-      width: '50rem',
-      px: 2,
-      pb: 4, // makes sure the button is visually inside the outer most box
-      borderRadius: 2,
-    }}>
-      <Typography variant="h3" className="centered-text" sx={{ py: 2, color: theme => theme.palette.common.white, }}>{`${employee.firstName} ${employee.lastName}`}</Typography>
-      <Container style={{ overflowY: 'scroll', maxHeight: '40rem' }} sx={{ borderRadius: 2, bgcolor: theme => theme.palette.common.white}}>
-        <Typography variant="h6" className="centered-text" sx={{ m: 2, }}>{employee.email}</Typography>
-        <Typography variant="h6" className="centered-text" sx={{ m: 2, }}>{employee.phoneNumber}</Typography>
+    <Box
+      sx={{
+        bgcolor: theme => theme.palette.primary.main,
+        height: 'fit-content',
+        m: '8rem auto 0 auto',
+        width: '50rem',
+        px: 2,
+        pb: 4, // makes sure the button is visually inside the outer most box
+        borderRadius: 2,
+      }}
+    >
+      <Typography variant="h3" className="centered-text" sx={{ py: 2, color: theme => theme.palette.common.white, }}>{`${employee?.firstName} ${employee?.lastName}`}</Typography>
```

Comment:
Consider something like:

```
import { styled } from '@mui/system';
import Typography from '@mui/material/Typography';

const StyledTypography = styled(Typography)({
  paddingY: 2,
  color: theme => theme.palette.common.white,
});

<StyledTypography variant="h3" className="centered-text">
  {`${employee?.firstName} ${employee?.lastName}`}
</StyledTypography>
```

Reply:
Consider making the fullName variable part of `useGetEmployeeByIdQuery` so it returns the fullName

`  const { data: employee } = useGetEmployeeByIdQuery(urlParams?.id)`